pinot-controller/src/main/java/org/apache/pinot/controller/api/resources/PinotSegmentUploadRestletResource.java

```diff
@@ -352,10 +352,10 @@ private String getZkDownloadURIForSegmentUpload(FileUploadPathProvider provider,
     if (baseDataDirURI.getScheme().equalsIgnoreCase(CommonConstants.Segment.LOCAL_SEGMENT_SCHEME)) {
       return URIUtils.constructDownloadUrl(provider.getVip(), rawTableName, segmentName);
     } else {
-      // Receiving .tar.gz segment upload for pluggable storage
-      LOGGER.info("Using configured data dir {} for segment {} of table {}", _controllerConf.getDataDir(), segmentName,
-          rawTableName);
-      return URIUtils.constructDownloadUrl(baseDataDirURI.toString(), rawTableName, segmentName);
+      // Receiving .tar.gz segment upload for pluggable storage. Download URI is the same as final segment location.
```

Comment:
final segment location -> deep store URI or segment store URI to make it clearer? The word final sounds a bit vague to me.